lib/kitchen/command.rb

```diff
@@ -170,17 +170,25 @@ def run_action(action, instances, *args)
         concurrency.times do
           threads << Thread.new do
             while instance = queue.pop
+              puts "running #{instance.name}"
```

Comment:
Is this debug output? If not, we should probably use the logger methods instead.